.github/workflows/ci.yml

```diff
@@ -0,0 +1,89 @@
+name: CI
+
+on:
+  push:
+    branches: [ jgrpp ]
+  pull_request:
+    branches: [ jgrpp ]
+
+jobs:
+  build_linux:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: prepare workspace
+      run: mkdir build
+    - name: install dependencies
+      run: "sudo apt-get install -y zlib1g-dev liblzo2-dev liblzma-dev libpng-dev libfreetype6-dev libfontconfig1-dev libicu-dev libsdl2-dev"
+    - name: cmake
+      run: |
+        pushd build
+        cmake ..
+        popd
+    - name: make package
+      run: |
+        pushd build
+        make package
+        popd
+    - name: upload artifact
+      uses: actions/upload-artifact@v2
+      with:
+        name: linux
+        path: build/bundles/*
+
+  build_macos:
+    runs-on: macos-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: prepare workspace
+        run: mkdir build
+      - name: brew update
+        run: "brew update"
+      - name: install dependencies
+        run: "brew install zlib lzo xz libpng freetype fontconfig icu4c sdl2"
```

Comment:
I was having some difficulties compiling a version of this for Enjineer a few months ago, brew seemed to be installing only dynamic libs. The executable works on my machine because I have the libraries installed, but I'm thinking this might not work for people who don't have these libraries installed. I have access to a non-brew'ed machine, so I'll give it a test later today.

Reply:
hm yeah, i'm not sure how to get homebrew to use static libs

Reply:
and i have a feeling the same issue will happen with linux